Sources/QueuesRedisDriver/JobsRedisDriver.swift

```diff
@@ -55,6 +57,26 @@ public struct RedisQueuesDriver {
     ///   - eventLoopGroup: The `EventLoopGroup` to run the driver with
     public init(configuration config: RedisConfiguration, on eventLoopGroup: EventLoopGroup) {
         let logger = Logger(label: "codes.vapor.redis")
+        let eventLoop = eventLoopGroup.next()
+        let redisTLSClient: ClientBootstrap? = {
+            guard let tlsConfig = config.tlsConfiguration,
+                    let tlsHost = config.tlsHostname else { return nil }
```

Comment:
A missing `tlsHost` should not cause immediate bailout here; it's valid in some configurations to have TLS enabled without it.
```suggestion
            guard let tlsConfig = config.tlsConfiguration else { return nil }
```